vm.image.js

```diff
@@ -126,9 +126,9 @@ Object.subclass('Squeak.Image',
             if (fileHeaderSize > 512) throw Error("bad image version");
         };
         this.version = version;
-        var nativeFloats = [6505, 6521, 68003, 68021].indexOf(version) >= 0;
-        this.hasClosures = [6504, 6505, 6521, 68002, 68003, 68021].indexOf(version) >= 0;
-        this.isSpur = [6521, 68021].indexOf(version) >= 0;
+        var nativeFloats = [6505, 6521, 68003, 68021, 68533].indexOf(version) >= 0;
+        this.hasClosures = [6504, 6505, 6521, 68002, 68003, 68021, 68533].indexOf(version) >= 0;
+        this.isSpur = [6521, 68021, 68533].indexOf(version) >= 0;
```

Comment:
I think these three lines would also need to recognize format 7033.

@dtlewis290 is there a better way to check these? Like testing a bit instead of explicitly listing every single version?

Reply:
Yes adding 7033 to those three lists should work. And yes it might better be done with bit testing. See ImageFormat class>>bitAssignments for reference. Bits 1, 5, and 10 can be tested in this manner.